frame/nomination-pools/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Runtime API definition for nomination-pools pallet.
+//! Currently supports only one rpc endpoint.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::Codec;
+
+sp_api::decl_runtime_apis! {
+	pub trait NominationPoolsApi<AccountId, Balance>
+		where AccountId: Codec, Balance: Codec
```

Comment:
For some reason, the ```NominationPoolsApi``` trait is not getting exposed. The implementation in companion cannot find this trait. 

cc: @kianenigma

Reply:
where are you running into this issue?

Reply:
When building westend and kusama runtimes locally and on the companion CI check(https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/1692761)

Reply:
Hmm yeah I think the companion system is not able to pickup new crates unfortunately. 

cc @paritytech-ci

Reply:
Okay, for now, let's not add the PRC endpoint anywhere and only add the new runtime api. This can already be called by the `state_call` RPC endpoint. I guess we can still keep the RPC crate, but let's not include it anywhere.